edb/lib/std/20-genericfuncs.edgeql

```diff
@@ -55,6 +55,16 @@ std::len(array: array<anytype>) -> std::int64
 # std::sum
 # --------
 
+CREATE FUNCTION
+std::sum(s: SET OF std::bigint) -> std::bigint
```

Comment:
You need to document all new signatures. Like in this case you should update `funcops/set.rst`.